ROMFS/px4fmu_common/init.d/airframes/13000_generic_vtol_standard

```diff
@@ -16,6 +16,7 @@
 # @output AUX3 Elevator
 # @output AUX4 Rudder
 # @output AUX5 Throttle
+# @output AUX6 Pusher reverse channel
```

Comment:
I would prefer to call it "Throttle reverse" - as it also can be a puller and not a pusher. You can also drop the "channel" here.

Reply:
Proposition accepted. Will change on the next commit.